pkgs/development/python-modules/django-filter/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, django
+, buildPythonPackage
+}:
+buildPythonPackage rec {
+
```

Since this package requires Python>=3.4, please add this.
```suggestion
broken = isPy2
```
You will also have to add `isPy2` as an argument :)